### ISSUE TITLE
Code the activePatrol method and test it

### ISSUE THREAD

**Issue by ashkan-software** [sid-devic/rddnn#1]
Code the activePatrol method and test it.
The activePatrol is our method for making a distributed neural network resilient, by making the highway connections' wights variable, and learn them using SGD.

**Comment by sid-devic** [comment]
I have implemented this method in the most recent commit. Here are some results, since it is a stochastic method I have done a few runs to get a better idea of how well it performs. The seeds are included to make the training reproducible.

|   Run	|  Validation set Acc  	|   Failure_Iteration acc	|  np_seed, tf_seed |
|---	|---	|---	|--- |
|   1	|   	98.4%| 90.479%  	|  (forgot to note down) | 	
|   2	|   95.6%	|   	89.710% |  2348, 2187 | 	
|   3	|   96.3%	|   90.851%	|   	7035, 3346 |
|   4	|   98.7%	|   84.1234%	|   5774, 2937 |
|   5	|   97.5%	|   89.1599%	|   8295, 7792 |

Closing, I see no drastic improvements to using this method versus simple networks of stochastic depth (activeGuard).
